src/Components/Profile/MyPosts/Post/post.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+import style from './post.module.css';
+
+import logo from './post.jpg';
+
+const Post = (props) => (
+  <div className={style.item}>
+    <img src={logo} alt="logo" />
+    {props.message}
```

Comment:
Just for your information (sometimes you can see FYI - "for your information"), the error that linter showed you about "destructuring assignment" was that you should default to get the fields out of the object the following way:
`const { message } = props; `

Reply:
Sure,I will be spent time to read information from vs code
![image](https://user-images.githubusercontent.com/46630057/74649901-962f2000-5191-11ea-8bda-b54ec6fe214a.png)